### DISCUSSION TITLE
Inputs with reset icon vs. global reset

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2987] [General] [answered]
Hi! @shannoncawley and I were taking a look at some editor designs that include the reset icon, and were wondering if there are any thoughts on best practices for using these on individual inputs vs one button that would reset all inputs in a build area (+ help text explaining that typing "auto" in an input will reset that input, but leave the others alone (?)). The reset icon is primary coupled with the color picker input, but we're also adding it on other width/height inputs, which can sometimes start to make the UI look a little cluttered:

![Screenshot 2023-01-26 at 3 56 04 PM](https://user-images.githubusercontent.com/87141551/214969219-54cb7f0b-4a8b-4879-bde7-92dd3cc6d2a4.png)

Just wanted to throw that out there, would appreciate any feedback. Thanks!

**Comment by gloriliale** [comment]
Thanks @ceyerkes! I'm bringing this up to our team and will get back to you soon!

**Comment by gloriliale** [comment, marked as answer]
Hello hello! Okay @ceyerkes, a few questions:
- Is the resetting capability necessary for this MVP? Is it to match the previous unlayer capabilities? 
- What do the inputs reset to? For background color for example, what would that be after hitting reset?
- Why are some fields "reset-able" and not others?

If you've found that the resetting capability is something you need in this panel, we'd recommend a global reset approach (one reset button that affects all these fields).

**Reply by ceyerkes** [reply]
Hellllloooo! 

- Yes, I think resetting is necessary for v1 in order to match current MC (cc: @shannoncawley) 
- Inputs reset to "auto" which is usually black/gray/white depending on what the input is. Happy to send through a Loom to make explaining a bit easier.
- I'm not sure what the logic is around what's resettable and what isn't. Maybe @cfree can give more context on that?

**Reply by cfree** [reply]
No clue! It seems arbitrary, but the hex code reset makes sense as most marketers likely won't have their hex codes memorized 😄

**Reply by loreina** [reply]
Thanks for the context! Our team would recommend going for a global reset approach then.
